src/Model/Document/ResultInterface.php

```diff
@@ -99,10 +99,11 @@ public function addTaxon(string $code, string $name, int $position, int $level,
      * @param string $channel
      * @param string $currency
      * @param int $value
+     * @param string[] $appliedPromotions
      *
      * @return ResultInterface
      */
-    public function addPrice(string $channel, string $currency, int $value): self;
+    public function addPrice(string $channel, string $currency, int $value, array $appliedPromotions = []): self;
```

Comment:
This looks like a BC break to me, since it forces anyone that implements this interface to update their method to add the new parameter. I'm not 100% sure only because the added parameter is optional.
In order to avoid the BC break, perhaps one could create a new interface that extends from this one and that redefines the method to add the new parameter.

Reply:
Usually adding an optional parameter is not considered a BC break. Besides, we are in RC tags, so we don't have a public API not to break. Finally, but not the less important, we are the only users of this fork probably 😆

Reply:
It's a BC break @lruozzi9. If someone implemented the interface he's forced to adapt the implementation otherwise it would result in a fatal error.

Reply:
Ah right, we are in an interface... I thought this method was inside the trait. I can check if there is a way to remove this parameter.

Reply:
Now should be ok.